packages/cli/src/commands/generate/scaffold/templates/components/NameForm.js.template

```diff
@@ -3,7 +3,7 @@ import {
   FormError,
   FieldError,
   Label,
-  ${fieldsToImport.join(',\n  ')},
+  ${fieldsToImport.length > 0 ? (fieldsToImport.join(',\n  ') + ',') : null}
```

Comment:
Personal preference really, but I'd write it like this
```javascript
${fieldsToImport.length && fieldsToImport.join(',\n  ') + ','}
```

And then I *think* we're actually passing it all through prettier.js. If we are you could remove the `\n  ` stuff.

```javascript
${fieldsToImport.length && fieldsToImport.join(',') + ','}
```

More important, however, than the comment above is if you could please write a test for this. Add a new model to `redwood/packages/cli/src/commands/generate/scaffold/__tests__/fixtures/schema.prisma` and a new fixture in `redwood/packages/cli/src/commands/generate/scaffold/__tests__/fixtures/components` then connect it all up in `scaffold.test.js`

Reply:
Sure thing on the length, honestly forget that works sometimes. 

And sure, I just left that as it was, but yeah if it's not needed then it can just be join() as the default is a comma.

And gotcha, tests are my weakness 😅

Reply:
> And sure, I just left that as it was, but yeah if it's not needed then it can just be join() as the default is a comma.

The tests will let us know! 😉

> And gotcha, tests are my weakness

Haha, don't worry, it's everyone's weakness 😛 Hopefully the Jest and Storybook integration in RW can change that for some of us!

Reply:
In the file schema.prisma in the path mentioned I've added a new model, I noticed above the model that it states ```// Define your own models here and run yarn db:save to create
// migrations for them.``` but that command doesnt exist, do I need to use the same I use in the client `yarn rw db save`?

Reply:
I'm kind of lost on the component if im honest. Since I need to basically just run the form, would I make another copy of form.js removing the imports and changing the name and just writing a test for that?